.github/workflows/ci-cd.yml

```diff
@@ -24,3 +24,9 @@ jobs:
         shell: pwsh
         run: |
           dotnet build .\Clean.Architecture.sln -c Release
+
+      - name: Run unit tests
+        id: run_unit_tests
+        shell: pwsh
+        run: |
+          dotnet test ./Clean.Architecture.sln --configuration Release
```

Comment:
1. Look at these two lines of code. It is unacceptable to have such difference. Unify how write them:
26 - `dotnet build .\Clean.Architecture.sln -c Release`
32 - `dotnet test ./Clean.Architecture.sln --configuration Release`

Hint: for slashes, use the way the terminal autocompletes the path to the solution
Hint 2: for cmdlet names, arguments, flags and parameters, when writing scripts the best practice is to use the full name e.g. `gc` is an alias for `Get-ChildItem`, so when using the terminal `gc` is perfectly fine, but when writing a script `Get-ChildItem` should be used

2. Add a `--no-build` flag to the `dotnet test` cmdlet, run the pipeline and describe under this review comment, if there are differences in the outcome or if the outcome is the same ([flag usage in `dotnet` cmdlet](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test))

Reply:
the only difference between a pipeline run with flag and without is that the "dotnet test" step does not execute restore and build.

Reply:
@szlatkow I know what the `--no-build` flag technically does. What I wanted you to describe is the outcome of the pipeline - does it break, does it have issues, does anything different happen or does it work as before and nothing is different ?

Reply:
It does not brake the "dotnet test" step and the difference is that thee build isn't executed for the step "dotnet test"